src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -1675,6 +1697,7 @@ void Eltwise::initSupportedPrimitiveDescriptors() {
 
     // if dim rank is greater than the maximum possible, we should use the reference execution
     canUseOptimizedImpl = mayiuse(x64::sse41) && getInputShapeAtPort(0).getRank() <= MAX_ELTWISE_DIM_RANK;
+    canUseOptimizedImpl &= !one_of(getAlgorithm(), Algorithm::EltwiseIsFinite, Algorithm::EltwiseIsInf, Algorithm::EltwiseIsNaN);
```

Comment:
We have to add JIT impls for all ops, Can be done in separate PR

Reply:
Appropriate ticket was created